### PR TITLE
feat(Image): add position prop

### PR DESCRIPTION
**Pull request by Jungzl**
RT

### CODE REVIEW

File: packages/vant/src/image/Image.tsx [hunk 1]
```diff
@@ -29,10 +29,13 @@ const [name, bem] = createNamespace('image');
 
 export type ImageFit = 'contain' | 'cover' | 'fill' | 'none' | 'scale-down';
 
+export type ImagePosition = 'center' | 'top' | 'right' | 'bottom' | 'left';
```
Comment (chenjiahan) [inline packages/vant/src/image/Image.tsx:32]: maybe should be `string` type to support `10px`

File: packages/vant/src/image/Image.tsx [hunk 2]
```diff
@@ -29,10 +29,13 @@ const [name, bem] = createNamespace('image');
...
+
 const imageProps = {
   src: String,
   alt: String,
   fit: String as PropType<ImageFit>,
+  position: String as PropType<ImagePosition>,
```
Comment (chenjiahan) [inline packages/vant/src/image/Image.tsx:38]: maybe should be `string` type to support `10px`

Comment (Jungzl) [reply packages/vant/src/image/Image.tsx:44]: just updated to align with `object-position`

**Comment by chenjiahan** [comment]
CI / test is failed, please run `npm run test:watch` and update the snaoshot~

**Comment by Jungzl** [comment]
done.

**Review by chenjiahan — APPROVED** [review]

**Comment by chenjiahan** [comment]
👍👍

**Pull request merged**